lib/screens/user_registration/user_registration.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:flutter/material.dart';
+import 'package:teesco/screens/login/widgets/password_field.dart';
+import 'package:teesco/screens/user_registration/widgets/email_input.dart';
+import 'package:teesco/screens/user_registration/widgets/institute_name_input.dart';
+import 'package:teesco/screens/user_registration/widgets/phone_number_field.dart';
+import 'package:teesco/screens/user_registration/widgets/confirm_password_field.dart';
+
+class UserRegistration extends StatelessWidget {
+  TextEditingController userNameController;
+  TextEditingController emailController;
+  TextEditingController passwordController;
+  TextEditingController confirmPasswordController;
+  TextEditingController phoneNumberController;
+  TextEditingController instituteNameController;
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Form(
+          child: DraggableScrollableSheet(builder: (context, scrollController) {
```

Comment:
Why do we need a `DraggableScrollableSheet` here isn't `SingleChildScrollView` enough?

Reply:
Also, it looks quite weird for a signup form to be a scrollable sheet.

Reply:
I got distracted by UI designs offered by Dribble. In the end, I decided to keep it simple maybe I just forgot to remove it. Would be careful larter.